components/sections/Navigation/index.tsx

```diff
@@ -71,34 +81,40 @@ const Navigation = () => {
 							href: '/blog/tags/all',
 							event: 'homepage_mainnav_blog',
 							text: 'Blog',
+							keyword: 'blog',
 						},
 						{
 							href: '/pricing',
 							event: 'homepage_mainnav_pricing',
 							text: 'Pricing',
+							keyword: 'pricing',
 						},
 						{
 							href: 'https://docs.shuttle.dev',
 							event: 'homepage_mainnav_docs',
 							text: 'Docs',
+							keyword: 'docs',
 						},
 						{
 							href: '/shuttle-heroes',
 							event: 'homepage_mainnav_heroes',
 							text: 'Shuttle Heroes',
+							keyword: 'heroes',
 						},
 						{
 							href: '/ai',
 							event: 'homepage_mainnav_ai',
 							text: 'Shuttle AI',
+							keyword: 'ai',
 						},
 						{
 							href: '/launchpad',
 							event: 'homepage_mainnav_launchpad',
 							text: 'Launchpad',
+							keyword: 'launchpad',
 						},
-					].map(({ event, href, text }) => (
-						<LinkItem key={href} event={event} href={href} text={text} setOpen={setOpen} />
+					].map(({ event, href, text, keyword }) => (
+						<LinkItem key={href} event={event} href={href} text={text} setOpen={setOpen} active={router.pathname.includes(keyword)} />
```

Comment:
It would be better to check if the pathname starts with `/blog` etc instead, since just looking for the keyword will sometimes match on unrelated paths.

Reply:
I was incorrect, pathname only includes path for the route file https://nextjs.org/docs/pages/api-reference/functions/use-router#router-object